openghg/openghg
# Adding new emissions keywords

Adding additional emissions keywords to be used for storing and allowing lookup based on these keywords. This includes:
 - database
 - database_version
 - model

These have also been added as keywords to the `openghg.retrieve.search_flux` and `openghg.retrieve.get_flux` functions

Tutorial added incorporating these details for adding flux, footprint and bc data
  - [doc/source/tutorials/local/Adding_new_data/Adding_ancillary_data.rst](https://github.com/openghg/openghg/blob/140a0b026d5b55eb87b869adce7b589521efaa90/doc/source/tutorials/local/Adding_new_data/Adding_ancillary_data.rst)

Tests added:
 - Check new metadata keywords can be added
 - Check object store entries can be selected and distinguished based on these keywords
 - If multiple datasources are found this fails nicely - (DatabaseLookupError with error message)

Closes #489 
Closes #446
